measure/src/main/scala/org/apache/griffin/measure/step/builder/dsl/transform/AccuracyExpr2DQSteps.scala

```diff
@@ -125,14 +126,20 @@ case class AccuracyExpr2DQSteps(context: DQContext,
       // 4. accuracy metric
       val accuracyTableName = ruleParam.getOutDfName()
       val matchedColName = details.getStringOrKey(_matched)
+      val matchedFractionColName = details.getStringOrKey(_matchedFraction)
```

Comment:
@bhlx3lyx7  
@chemikadze 
has this  broken Apache Griffin UI side? Not sure how UI consume from ES.

Reply:
As I know, UI calculates the matched fraction by total and matched count, in ES the metrics are just persisted in JSON format, it's OK to add a field here.